kubernetes/loculus/values.yaml

```
@@ -1441,68 +1440,221 @@ defaultOrganisms:
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/sars-cov-2/ORF9b.fasta]]"
         - name: "S"
           sequence: "[[URL:https://corneliusroemer.github.io/seqs/artefacts/sars-cov-2/S.fasta]]"
-  dummy-organism-with-files:
+  evs:
+    <<: *defaultOrganismConfig
     schema:
-      image: "https://cdn.who.int/media/images/default-source/mca/mca-covid-19/coronavirus-2.tmb-1920v.jpg?sfvrsn=4dba955c_19"
-      organismName: "Test organism (with files)"
-      nucleotideSequences: []
-      submissionDataTypes:
-        consensusSequences: false
-        files:
-          enabled: true
-          categories:
-            - name: raw_reads
-      files:
-        - name: raw_reads
-      metadata:
-        - name: date
-          type: date
+      <<: *schema
+      organismName: "Enterovirus"
+      nucleotideSequences: [CV-A16, CV-A10, EV-A71, EV-D68]
+      image: "/images/organisms/enterovirus.jpg"
+      linkOuts:
+        - name: "Nextclade (CV-A16)"
+          url: "https://clades.nextstrain.org/?input-fasta={{[unalignedNucleotideSequences+rich:CV-A16|fasta]}}&dataset-name=community/hodcroftlab/enterovirus/enterovirus/linked/CV-A16&dataset-server=https://raw.githubusercontent.com/alejandra-gonzalezsanchez/nextclade_data/multi-pathogen-evs/data_output"
```

could you try to invert this order? 
```suggestion
          url: "https://clades.nextstrain.org/?input-fasta={{[unalignedNucleotideSequences:CV-A16+rich|fasta]}}&dataset-name=community/hodcroftlab/enterovirus/enterovirus/linked/CV-A16&dataset-server=https://raw.githubusercontent.com/alejandra-gonzalezsanchez/nextclade_data/multi-pathogen-evs/data_output"
```